language/stdlib/modules/libra_account.move

```diff
@@ -147,7 +146,7 @@ module LibraAccount {
             Libra::approx_lbr_for_value<Token>(deposit_value) >= travel_rule_limit;
         // travel rule only applies if the sender and recipient are both VASPs
         let both_vasps = VASP::is_vasp(sender) && VASP::is_vasp(payee);
-        if (above_threshold &&
+        if (!0x0::Testnet::is_testnet() && above_threshold &&
```

Comment:
I think we actually want to have travel rule checking on in testnet. Otherwise, it will difficult for VASPs to test their off-chain logic...

Reply:
You make a good point. My only concern is that for "normal" testnet operations that this type of checking could be super annoying. But happy to remove it :)